buildSrc/src/main/kotlin/code-generation.gradle.kts

```diff
@@ -291,14 +301,51 @@ fun configurePublication(
 fun configurePom(
     mavenPublication: MavenPublication,
     implementationDependency: String,
+    providerName: String,
+    version: String,
 ) {
     mavenPublication.pom {
+        name.set("Pulumi ${providerName.capitalized()} Kotlin")
+        description.set("The Pulumi ${providerName.capitalized()} Kotlin SDK")
+        url.set("https://github.com/VirtuslabRnD/pulumi-kotlin")
+        inceptionYear.set("2022")
+
+        issueManagement {
+            system.set("GitHub")
+            url.set("https://github.com/VirtuslabRnD/pulumi-kotlin/issues")
+        }
+
         licenses {
             license {
                 name.set("The Apache License, Version 2.0")
                 url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
             }
         }
+
+        developers {
+            developer {
+                name.set("Dariusz Dzikon")
+                email.set("ddzikon@virtuslab.com")
+                organization.set("VirtusLab")
+            }
+            developer {
+                name.set("Michał Fudała")
```

Comment:
(minor)

I'd be in favour of only using basic ASCII characters here. (`ł` -> `l`)